form/src/main/java/org/dhis2/form/ui/FormView.kt

```diff
@@ -733,32 +732,35 @@ class FormView : Fragment() {
             .show()
     }
 
-    private fun showDescriptionLabelDialog() {
-        ComposeDialogFragment()
-            .show(childFragmentManager, "ComposeDialog")
-    }
+    private fun showDialog(intent: RecyclerViewUiEvents.ShowDescriptionLabelDialog) {
+        val dataElement = getD2().dataElement(intent.uid)
```

Comment:
intent.uid is enough to compare, it holds the id, don't need go get the entire dataElement.